### ISSUE TITLE
Isn't this actually whitespace?

### ISSUE THREAD

**Issue by arafangion** [kelseyhightower/nocode#116]
I think you did actually write code using whitespace. 

In particular, you have the following code, in whitespace:

```

```

I think this is unintentional, this code should be removed.

* Whitepace: https://en.wikipedia.org/wiki/Whitespace_%28programming_language%29

**Comment by 67ideas** [comment]
I agree completely. Not only should NoCode not involve code of its own -- it should also never lead to the accidental running of _other people's code_. In this case, some code process somewhere was involved in rendering the following markup:

`<pre><code></code></pre>`

That's inexcusable. I vote to remove the README and all associated files that might have unintentional side effects like the above.
